ex03.ex

```diff
@@ -101,7 +121,36 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([],[]) do
```

Comment:
# 3

Aarrgh... A little piece of my soul just died.

Never, ever, test booleans against true or false.

And when you find yourself writing functions longer than a few lines, stop and think.

Here's an alternative:

~~~ elixir
  def list_equal([], []), do: true
  def list_equal([h | tail1], [h | tail2]), do: list_equal(tail1, tail2)
  def list_equal(_, _), do: false
~~~